src/FeedGenerator.php

```diff
@@ -128,6 +128,8 @@ private function start_generation() {
 	 */
 	protected function handle_start() {
 		self::log( __( 'Feed generation start. Preparing temporary files.', 'pinterest-for-woocommerce' ) );
+		/* Remember the time a feed generation has started. */
```

Comment:
Please use single line comments for singular sentences.

In general I would omit comments like this. Thats because you have used a very descriptive name.